### PR TITLE
removed unwanted para tags in the blog articles

### PR DESCRIPTION
**Pull request by sashikanth1994**
Addresses #193 

#### What is included in this PR?
Remove stray closing p tags
Confirmed that the changed pages render
image

### CODE REVIEW

File: _posts/2011-02-16-an-odk-success-story-from-afghanistan.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ aktt_notify_twitter:
 ---
 We just received a note from David Hinkle, Director of [Afghanistan Infrastructure Data Center](http://www.aidc.af/aidc/) (AIDC) that we wanted to share. AIDC tracks and reports on civilian and military donor infrastructure projects in order to provide USAID and the supported Islamic Republic of Afghanistan (IRoA) ministries with a comprehensive and accurate picture of infrastructure development and related security incidents.
 
-David wrote, “_As an NGO performing infrastructure development oversight activities under USAID funding we have struggled with collecting accurate reports from field monitors located in remote and insecure parts of the country. </p>
```
Comment (danbjoseph) [inline _posts/2011-02-16-an-odk-success-story-from-afghanistan.md]: the underscore at the start of this quote can also be removed.

File: _posts/2011-05-24-mhealth-solution-with-odk-and-drupal-7.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ aktt_notify_twitter:
 ---
 Sanjay Patel and Adam Hammouda from [WebFirst](http://webfirst.com/) will be at University of Maryland’s [HCI Lab Symposium](http://www.cs.umd.edu/hcil/soh/) this week, presenting on their work on a [Open Source mHealth Solution based on Drupal 7](http://www.cs.umd.edu/hcil/sharp/workshop2011/speakers.html). The abstract is below.
 
-“_Many data collection platforms exist in the world of mobile health. Examples of these include: EpiSurveyor, Ushahidi, MOTECH, RapidSMS, FrontlineSMS. While these solutions are low-cost and effective for many mHealth interventions, they are limited in their flexibility, scalability, and ability to publish and share data. In order to fully achieve the goals of the mHealth community (improving access to information, improve data reliability, improve the quality of care by providing better decision-support tools) it is important to consider platforms that are open and well-supported by a large developer community.</p>
```
Comment (danbjoseph) [inline _posts/2011-05-24-mhealth-solution-with-odk-and-drupal-7.md]: underscore at start can be removed here, and the `</em>` at the end of line 20.

File: _posts/2011-07-26-experiences-with-kobo-and-odk-in-bolivia.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ aktt_notify_twitter:
 
 Maryann used the [KoBo Project](http://koboproject.org/), data collection tools from Berkeley’s [Human Rights Center](http://www.law.berkeley.edu/HRCweb/index.html). Like ODK, KoBo is built on the [OpenRosa](http://openrosa.org/) standard. The popular tools are the KoBo Form Builder (based on [PurcForms](http://code.google.com/p/purcforms/)) and KoBo Post Processor. ODK and KoBo tools are fully compatible so many users like Maryann can mix and match as they need!
 
-She writes, _“I was lucky enough to have the funding to purchase 5 Huawei IDEOS devices and worked with ODK and KoBo for my surveys. </p> 
+She writes, _“I was lucky enough to have the funding to purchase 5 Huawei IDEOS devices and worked with ODK and KoBo for my surveys.
```
Comment (danbjoseph) [inline _posts/2011-07-26-experiences-with-kobo-and-odk-in-bolivia.md]: underscore at the start here can be removed, and the `</em>` at the end of line 54.

File: _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ aktt_notify_twitter:
...
 
-He writes, _“I’m putting a call out on behalf of an organization trying to use ODK and related technology to help respond to the unfolding famine in East Africa. The Royal Veterinary College is supporting Vetaid Kenya and other Vetaid partners in the South East of the country – where some grazing and food stocks remain – to try and help the pastoralists breeding stock alive through the next 6 months. Obviously the priority has rightly been feeding the people first but without their livestock there is little chance of hem ever leaving the camps and returning to their pastoral way of life.</p> 
+He writes, _“I’m putting a call out on behalf of an organization trying to use ODK and related technology to help respond to the unfolding famine in East Africa. The Royal Veterinary College is supporting Vetaid Kenya and other Vetaid partners in the South East of the country – where some grazing and food stocks remain – to try and help the pastoralists breeding stock alive through the next 6 months. Obviously the priority has rightly been feeding the people first but without their livestock there is little chance of hem ever leaving the camps and returning to their pastoral way of life.
```
Comment (danbjoseph) [inline _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: underscore at the start here on line 14 can be removed, and the `</em>` at the end of line 24.

Comment (sashikanth1994) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: Those underscores are used to make the text italics.
Do you still want me to remove them?

Comment (sashikanth1994) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: Removed the </em> tag across files

Comment (danbjoseph) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: When it is a matched pair, I would leave them. But many are single and render on the page:
Screen Shot 2019-08-01 at 2 39 10 PM

Thank you for your continued work helping to clean this up!

Comment (danbjoseph) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: This particular paragraph could also use a `"` at the end to close the quotation from the man's writing.

File: _posts/2011-02-16-an-odk-success-story-from-afghanistan.md (hunk 1 above)
Comment (danbjoseph) [reply _posts/2011-02-16-an-odk-success-story-from-afghanistan.md]: if everything that "David wrote" is supposed to be rendered in italics, then that requires edits as well. See the page: https://opendatakit.org/2011/02/an-odk-success-story-from-afghanistan/

File: _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md (hunk 4 above)
Comment (sashikanth1994) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: So, Should we remove the single underscore or should we pair all, to render them on website in italics?

Comment (danbjoseph) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: I don't think a post should be almost entirely in italics, so I'm in favor of removing.

Comment (sashikanth1994) [reply _posts/2011-08-03-urgent-need-for-odk-assistance-east-africa-famine.md]: As a way to differentiate the customer quotes from the body of the post, I have 'blockquote'd it. The result looks like in the attached image - Suggestion_01.

However I noticed that the blockquote CSS in the main.css sets the style to italics, causing the entire comment to be rendered in italics.

Suggestion_01

Let me know your thoughts.

**Comment by yanokwa** [comment]
Closing because it's no longer relevant. Repo is now at https://github.com/DSFSG/opendatakit.org in the unlikely case someone wants to move these edits over.

**Pull request closed without merging**